### PR TITLE
WEBUI-1099: Fix how vocabulary entries are deleted

### PR DESCRIPTION
**Pull request by poonamyadav252**

### CODE REVIEW

File: packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/vocabularies.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ After(() =>
   Promise.all(
     addedVocabularyEntries.map((entry) =>
       nuxeo
-        .request(`/directory/${entry.directoryName}`)
+        .request(`/directory/${entry.directoryName}/${entry.id}`)
         .delete(entry)
```
Comment (Nisha-QA) [inline packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/vocabularies.js]: Please make below changes  as per Kevin's PR
.request(`/directory/${entry.directoryName}/${entry.id}`)
        .delete()

Comment (poonamyadav252) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/vocabularies.js]: I have taken reference from Kevin's only, then what should I need to change?

PR references: PR #1780 and #1781

Comment (Nisha-QA) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/vocabularies.js]: Check line 33

Comment (poonamyadav252) [reply packages/nuxeo-web-ui-ftest/features/step_definitions/support/fixtures/vocabularies.js]: Done

**Review by Nisha-QA — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by satishyadav-dev — APPROVED** [review]

**Pull request merged**